plugins/containers-plugin/src/containers-tree-data-provider.ts

```diff
@@ -39,14 +42,25 @@ export class ContainersTreeDataProvider implements theia.TreeDataProvider<ITreeN
 
         this.updateContainersTreeData(containersService).then(() => {
             this.onDidChangeTreeDataEmitter.fire();
+        }, error => {
+            console.error(error);
```

Comment:
does it need to be reported to user or not ?

Reply:
Good idea. Done